roles/confluent.control_center/defaults/main.yml

```diff
@@ -19,12 +19,14 @@ control_center_rocksdb_path: ""
 control_center_service_overrides:
   User: "{{ control_center_user if control_center_user != control_center_default_user else '' }}"
   Group: "{{ control_center_group if control_center_group != control_center_default_group else '' }}"
+  ExecStart: "{% if installation_method == 'archive' %}{{ control_center.server_start_file }} {{ control_center.config_file }}{% endif %}"
 
 control_center_service_environment_overrides:
   ROCKSDB_SHAREDLIB_DIR: "{{control_center_rocksdb_path}}"
   CONTROL_CENTER_HEAP_OPTS: "-Xmx6g"
   CONTROL_CENTER_OPTS: "{{ control_center_final_java_args | java_arg_build_out }}"
   CONTROL_CENTER_LOG4J_OPTS: "{% if control_center_custom_log4j|bool %}-Dlog4j.configuration=file:{{control_center.log4j_file}}{% endif %}"
+  LOG_DIR: "{{control_center.appender_log_path}}"
```

Comment:
is LOG_DIR something that was already there in the server file that we missed?

BTW this is a perfect use of these service override dicts!

Reply:
Yes, it is in the original.